src/NzbDrone.Core/Notifications/Kavita/Kavita.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Sockets;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Extensions;
+
+namespace NzbDrone.Core.Notifications.Kavita;
+
+public class Kavita : NotificationBase<KavitaSettings>
+{
+    private readonly IKavitaService _kavitaService;
+    private readonly Logger _logger;
+
+    public Kavita(IKavitaService kavitaService, Logger logger)
+    {
+        _kavitaService = kavitaService;
+        _logger = logger;
+    }
+
+    public override string Link => "https://www.kavitareader.com/";
+
+    public override void OnReleaseImport(BookDownloadMessage message)
+    {
+        var allPaths = message.BookFiles.Select(v => v.Path).Distinct();
+        var path = Directory.GetParent(allPaths.First())?.FullName;
+        Notify(Settings, "Readarr - Downloaded", path);
+    }
+
+    public override void OnBookDelete(BookDeleteMessage deleteMessage)
+    {
+        var allPaths = deleteMessage.Book.BookFiles.Value.Select(v => v.Path).Distinct();
+        var path = Directory.GetParent(allPaths.First())?.FullName;
+        Notify(Settings, "Readarr - Book Deleted", path);
+    }
+
+    public override void OnBookFileDelete(BookFileDeleteMessage message)
+    {
+        Notify(Settings, "Readarr - Book File Deleted", Directory.GetParent(message.BookFile.Path)?.FullName);
```

Comment:
Do we have pre-defined variables for these? Similar to what you used at L45 (BOOK_RETAGGED_TITLE_BRANDED)

Reply:
Got it, will apply across the code.